_tutorials/new-component/create-a-new-component-part-1.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Create A New Origami Component
+description: A step-by-step tutorial which teaches you how to build and deploy a new Origami component.
+cta: Learn how to create an Origami component
+
+---
+
+# {{page.title}}
+
+The "Create A New Origami Component" tutorial is split into eight parts and is intended to be followed sequentially from start to finish:
+- part one: Intro & Boilerplate
+- [part two](/docs/tutorials/create-a-new-component-part-2/): Base Styles
+- [part three](/docs/tutorials/create-a-new-component-part-3/): Themes & Brands
+- [part four](/docs/tutorials/create-a-new-component-part-4/): Demos
+- [part five](/docs/tutorials/create-a-new-component-part-5/): JavaScript
+- [part six](/docs/tutorials/create-a-new-component-part-6/): Testing
+- [part seven](/docs/tutorials/create-a-new-component-part-7/): Documentation
+- [part eight](/docs/tutorials/create-a-new-component-part-8/): Component Lifecycle
+
+## Introduction
+
+Usually the Origami team will own an Origami component which is used by multiple teams or groups, but anybody can create and share an Origami component.
+
+In this tutorial we'll build an Origami component. Our example component will display a "hello world" message and include a button which will count the number of times it was clicked. Visually the component will include multiple variations to suit reader facing and internal projects. We'll also discuss how to document and publish an Origami component so it is visible in the [Origami Registry](https://registry.origami.ft.com/components).
```

Comment:
>reader facing and internal projects
I've never seen the term "reader facing" before, do we mean master brand? If we mean master brand and we think "reader facing" is useful to those who don't know what master brand means, we could put master brand in brackets so that the user knows to relate those terms together.

>reader facing (master brand) and internal projects (internal brand).